src/product-categories/entities/product-category.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { ObjectType, Field, Int } from '@nestjs/graphql';
+import {Column, Entity, PrimaryGeneratedColumn} from "typeorm";
+
+@ObjectType()
+@Entity({ name: 'product-category' })
+export class ProductCategory {
+  @Field(() => Int)
+  @PrimaryGeneratedColumn()
+  Id: number;
```

Comment:
Nitpick: typically in JS, it is usually to start property names with lower case.

Reply:
I personally use camelCase, but in this case, I wanted it to be compatible with field names, so I started names with capital letter